### PR TITLE
Mostly comments and name changes

### PR DESCRIPTION
**Pull request by mattcushman**
* comments
* couple of variable name changes
* minor optimization of treeSum()

### CODE REVIEW

File: src/libraries/Deposits.sol [hunk 1]
```diff
@@ -150,20 +178,25 @@ library Deposits {
                     // Apply scaling to all range parents less then starting node, index_
                     deposits_.scaling[index_] = scaledFactor;
                 } else {
+                    // this node's scale factor is 1
                     sum += Maths.wmul(factor_, value) - value;
                     deposits_.scaling[index_] = factor_;
                 }
-
+                // Uset the bit in index to continue traversing up the Fenwick tree
```
Comment (grandizzy) [inline src/libraries/Deposits.sol]: `Uset` typo

**Review by grandizzy — APPROVED** [review]
Looks good, minor typo

File: src/libraries/Deposits.sol [hunk 2]
```diff
@@ -44,43 +60,54 @@ library Deposits {
...
+     *  @return sumIndexScale_ Scale of bucket PRECEDING sumIndex_
      */
     function findIndexAndSumOfSum(
         DepositsState storage deposits_,
         uint256 targetSum_
     ) internal view returns (uint256 sumIndex_, uint256 sumIndexSum_, uint256 sumIndexScale_) {
+        // i iterates over bits from MSB to LSB.  We check at each stage if the target sum is to the left or right of sumIndex_+i
         uint256 i  = 4096; // 1 << (_numBits - 1) = 1 << (13 - 1) = 4096
-        uint256 sc = Maths.WAD;
+        uint256 runningScale = Maths.WAD;
```
Comment (EdNoepel) [inline src/libraries/Deposits.sol:72]: 🙏

File: src/libraries/Deposits.sol [hunk 3]
```diff
@@ -259,9 +314,10 @@ library Deposits {
     function treeSum(
         DepositsState storage deposits_
     ) internal view returns (uint256) {
+        // In a scaled Fenwick tree, sum is at the root node, but needs to be scaled
         uint256 scaling = deposits_.scaling[SIZE];
-        if (scaling == 0) scaling = Maths.WAD;
-        return Maths.wmul(scaling,deposits_.values[SIZE]);
+        // scaling == 0 means scale factor is actually 1
+        return (scaling != 0) ? Maths.wmul(scaling,deposits_.values[SIZE]) : deposits_.values[SIZE];
```
Comment (EdNoepel) [inline src/libraries/Deposits.sol]: nit: missing space after `,` between `Maths.wmul` params

**Review by EdNoepel — APPROVED** [review]

File: src/libraries/Deposits.sol (hunk 1 above)
Comment (mattcushman) [reply src/libraries/Deposits.sol]: fixed

File: src/libraries/Deposits.sol (hunk 3 above)
Comment (mattcushman) [reply src/libraries/Deposits.sol]: fixed

**Pull request merged**